Ds-Session-4/employee/EmployeeTest.java

```diff
@@ -0,0 +1,179 @@
+package employee;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class EmployeeTest
+{
+	/**
+	 * test case for adding employees in SortByNaturalOrder class object, it rejects employees with duplicate id 
+	 */
+	@Test
+	public void sortByNaturalOrderTestFirst()
+	{
+		SortByNaturalOrder collection = new SortByNaturalOrder();
+
+		assertTrue(collection.addEmployee(new Employee(19, "Prakalpa", "delhi")));
+		assertTrue(collection.addEmployee(new Employee(25, "Deekshika", "alwar")));
+		assertFalse(collection.addEmployee(new Employee(19, "Anshul", "gurgaon")));
+		assertTrue(collection.addEmployee(new Employee(27, "Shalu", "delhi")));
+		assertFalse(collection.addEmployee(new Employee(25, "Disha", "mumbai")));
+		assertTrue(collection.addEmployee(new Employee(17, "arjita", "gwalior")));
+		assertTrue(collection.addEmployee(new Employee(3, "Tanu", "jaipur")));
```

Comment:
Above lines are written in many of the test methods, a common method should be used to avoid the repetitive work.